frontend/src/metabase/qb/lib/actions.js

```diff
@@ -40,6 +40,11 @@ export const toUnderlyingRecords = (card: CardObject): ?CardObject => {
     }
 };
 
+export const nestThisQuery = (card: CardObject): ?CardObject => {
+    const newCard = startNewCard("query", -1337, "card__" + card.id);
```

Comment:
also not sure how I feel about hardcoding `"card__"` like that, to do this the RightWay™ we would maybe consider adding a property to Cards that come back from the backend like `nested_query_id` or something that you can use instead of constructing the pseudo ID yourself